docs/marionette.region.md

```diff
@@ -261,6 +261,7 @@ In addition to adding and removing regions there are a few
 methods to help utilize regions.
 
 - `getRegion(name)` - Request a region from a view by name.
+ - Note: If the view hasn't been rendered at this point, it will be.
 - `getRegions()` - Returns an object literal of all regions on the view organized by name.
```

Comment:
@paulfalgout - part of view destruction includes `removeRegions`, which will then `getRegions` and re-render the view.  Consider this:

```js
const foo = new View({
   regions: {
      blah: '.foo'
   }
});
foo.destroy();
```

Do you see an issue with `destroy()` rendering the view AFTER the UI elements are unbound, and the view el is removed?  

```
destroy(...args) {
    if (this._isDestroyed) { return this; }
    const shouldTriggerDetach = !!this._isAttached;

    this.triggerMethod('before:destroy', this, ...args);
    if (shouldTriggerDetach) {
      this.triggerMethod('before:detach', this);
    }

    // unbind UI elements
    this.unbindUIElements();

    // remove the view from the DOM
    this.removeEl(this.el);

    if (shouldTriggerDetach) {
      this._isAttached = false;
      this.triggerMethod('detach', this);
    }

    // VIEW RENDERED HERE <-------------------------------------------
    this._removeChildren();

    this._destroyBehaviors(args);

    this._isDestroyed = true;
    this._isRendered = false;
    this.triggerMethod('destroy', this, ...args);

    this.stopListening();

    return this;
  },
```